spec/features/write_to_councillors_spec.rb

```diff
@@ -160,11 +160,10 @@
       open_email("louise@council.nsw.gov.au")
 
       expect(current_email).to_not have_content("For the attention of the General Manager / Planning Manager / Planning Department")
-      expect(current_email).to have_content("Hi Louise Councillor")
```

Comment:
It's actually come out of the current draft, so the test was updated to match, but I think it's probably important to have it there.

Having the name confirms that it's for them and further personalises it.

It also provides some help if someone other than the intended person is looking at it. After looking at more of the email addresses, I'd say it's common for the incomings get forwarded around to a few people, particularly the major@ addresses.